src/chromeDebugAdapter.ts

```diff
@@ -172,10 +172,23 @@ export class ChromeDebugAdapter extends CoreDebugAdapter {
 
     protected doAttach(port: number, targetUrl?: string, address?: string, timeout?: number, websocketUrl?: string, extraCRDPChannelPort?: number): Promise<void> {
         return super.doAttach(port, targetUrl, address, timeout, websocketUrl, extraCRDPChannelPort).then(() => {
+            const userAgentPromise = this.globalEvaluate({ expression: 'navigator.userAgent', silent: true }).then(evalResponse => evalResponse.result.value);
```

Comment:
There's an CDP api for this: https://chromedevtools.github.io/devtools-protocol/tot/Browser#method-getVersion

Reply:
Thanks, I didn't know that existed.